schema/test/unit/function_checksum_form_results_test.sql

```diff
@@ -0,0 +1,83 @@
+set client_min_messages to warning;
+create extension if not exists pgtap;
+reset client_min_messages;
+
+begin;
+select plan(4);
+
+select has_function(
+  'ggircs_portal', 'checksum_form_results',
+  'Function checksum_form_results should exist'
+);
+
+-- Insert organisation & facility test data
+insert into ggircs_portal.organisation(operator_name) values ('test org');
+insert into ggircs_portal.facility(organisation_id, facility_name) values (1, 'test facility');
```

Comment:
there's already organisations and facilities in the database (inserted by the deploy-data.sh script), so you don't need those